### DISCUSSION TITLE
add firewall rules to groups

### DISCUSSION THREAD

**Discussion by AveragePaintEnjoyer** [sophos/sophos-firewall-sdk#77] [General]
is it possible to add firewall rules to groups?
I like the idea of using groups to organize rules...

**Comment by mamullen13316** [comment]
This feature has been implemented in v0.1.59:

https://sophosfirewall-python.readthedocs.io/en/latest/firewallapi.html#firewallapi.SophosFirewall.create_rulegroup
https://sophosfirewall-python.readthedocs.io/en/latest/firewallapi.html#firewallapi.SophosFirewall.update_rulegroup
https://sophosfirewall-python.readthedocs.io/en/latest/firewallapi.html#firewallapi.SophosFirewall.get_rulegroup
